frontend_tests/puppeteer_tests/navigation.ts

```diff
@@ -97,7 +97,6 @@ async function navigation_tests(page: Page): Promise<void> {
     await navigate_to_subscriptions(page);
     await navigate_using_left_sidebar(page, "all_messages", "message_feed_container");
     await navigate_to_settings(page);
-    await navigate_using_left_sidebar(page, "narrow/is/private", "message_feed_container");
```

Comment:
You should replace this with another way to open Private Messages. Clicking on `#toggle-private-messages-section` should do the trick, and you could keep `message_feed_container` as the tag you're looking to be visible, though it would be nice to also check that the PM box expands into the list.

Sort of similar to what you already did in `message-basics.ts` :D

Reply:
@jai2201 this is still remaining